cmd/backupdriver/main.go

```diff
@@ -115,9 +128,5 @@ func newK8sClient(master, kubeConfig string) (kubernetes.Interface, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to create config: %v", err)
```

Comment:
Can we use errors.Wrapf() here?  Or we can start to use fmt.Errorf with %w and start a task to switch the rest of the code base over.

Reply:
Sure.

Reply:
Fixed.